backend/user_management/minio/schemas.py

```
@@ -0,0 +1,12 @@
+
+
+from pydantic import BaseModel
+from typing import List
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)